sdk/cmake/CMakeMacros.cmake

```diff
@@ -287,10 +287,10 @@ function(add_cd_file)
     endif()
 
     # do we add it to all CDs?
-    list(FIND _CD_FOR all __cd)
+    list(FIND _CD_FOR "all" __cd)
     if(NOT __cd EQUAL -1)
-        list(REMOVE_AT _CD_FOR __cd)
```

Comment:
So should this have just been `${__cd}` instead?
I recall this function being quite performance sensitive, so I'm not sure if having it repeat the search will be problematic.

Reply:
This list is small in general, moreover for all occurrences of "all" parameter, the list contains only one element (that actually explains how it worked all the time).
So I don't think it introduces any performance impact. We can probably just construct a new list here